domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/Exceptions.kt

```diff
@@ -15,8 +15,8 @@ class InductionAlreadyExistsException(val prisonNumber: String) :
 /**
  * Thrown when an Attempt is made to create an Induction Schedule for a prisoner who already has one.
  */
-class InductionScheduleAlreadyExistsException(val prisonNumber: String) :
-  RuntimeException("An Induction Schedule already exists for prisoner $prisonNumber")
+class InductionScheduleAlreadyExistsException(val inductionSchedule: InductionSchedule) :
```

Comment:
The reason I am passing in the entire InductionSchedule into the exception is that we will need it in the next PR.

Basically once the prisoner.admission listener starts to orchestrate the calls to the service methods, it will need to know if the prisoner already has an Induction Schedule (by catching the exception), but it will also need to know the status that the Induction Schedule is currently in, in order for it to set a new status & deadline date etc